hayk.poghosyan/repo/Lesson_3/exercise_08/exercise_08.py

```diff
@@ -0,0 +1,7 @@
+# 8
+number_1 = int(input("Enter a number"))
+number_2 = int(input("Enter a number"))
+number_3 = int(input("Enter a number"))
+total = number_1 + number_2 + number_3
+average = total/3
+print(total,"\n",average)
```

Comment:
CSW: Print the message in a proper way so that the user can see what presents each value.